modules/theme/supported_themes.nix

```diff
@@ -128,4 +128,30 @@
     '';
     styles = ["dark" "light"];
   };
+  rose-pine = {
+    setup = {
+      style ? "main",
+      transparent ? false,
+    }: ''
+      require("rose-pine").setup({
+        dark_variant = "${style}", -- main, moon, or dawn
+        dim_inactive_windows = false,
+        extend_background_behind_borders = true,
+
+        enable = {
+          terminal = true,
+          migrations = true,
+        },
+
+        styles = {
+          bold = false,
+          italic = false, -- I would like to add more options for this
```

Comment:
you're more than welcome to do so! I can't think of a backwards-compatible way of, but we can introduce a freeform `settings` attribute to be shared scross modules so that they are able to freely define their own settings :)